tsdb/engine/tsm1/engine.go

```diff
@@ -368,7 +369,7 @@ func (e *Engine) WritePoints(points []models.Point) error {
 }
 
 // DeleteSeries deletes the series from the engine.
```

Comment:
Or... Keep `DeleteSeries` and add `DeleteSeriesRange` that takes in the time range. Similar to the changes in the cache.